docs/reference/modules/admin-bar.md

```diff
@@ -0,0 +1,125 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/admin-bar`
+
+**Alias:** `apos.adminBar`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module implements Apostrophe's admin bar at the top of the page. Any module can register a button (or more than one) for the bar by calling this module's `add` method. Buttons can also be grouped into dropdown menus and restricted to those with particular permissions. The [apostrophe-pieces](/reference/modules/piece-type.md) takes advantage of this module automatically.
+
+## Configuration options
+
+Options are passed into the admin-bar module by "extending" it through an `index.js` file typically located in the project modules folder, nested inside an `admin-bar` subfolder, inside a `@apostrophecms` folder.
+For example
+```
+deployment
+lib
+modules
+┗ @apostrophecms
+┆              ┗ admin-bar
+┆                        ┗ i18n
+┆                        ┆    ┗ en.json
+┆                        ┆      fr.json
+┆                          index.js 👈
+@node_modules
```

Comment:
There is no `@` on `node_modules`

Reply:
I've removed the directory illustration. I guess the location verbiage above is good enough. I just wanted to emphasize that it is in a project-specific `@apostrophecms` folder. This is something that tripped me up reading the documentation.